timetracker/entries/models.py

```diff
@@ -14,8 +14,8 @@ class Project(models.Model):
     client = models.ForeignKey('Client', blank=True, null=True)
     name = models.CharField(max_length=200)
 
-    def __str__(self):
-        return '<{}> {}'.format(self.client, self.name)
+    def __unicode__(self):
+        return '{}'.format(self.name)
```

Comment:
You can just return `self.name` here. Django is using unicode everywhere, you don't need to cast it yourself.